wiki/Tournaments/CWC/1/en.md

```diff
@@ -157,91 +157,322 @@ This competition has come to an end and resulted in the following podium:
+10. Exchanging players during a match is allowed without limitations.
+    - **If a map rematch is required, exchanging players is not allowed. With the Tournament Management's discretion, an exception can be made if the previous roster is unavailable to play.**
+11. Lag is not a valid reason to nullify a beatmap.
+12. All players are supposed to keep the match running fluently and without delays. Penalties can be issued to the players if they cause excessive match delays.
+13. If a player disconnects between maps and the team cannot provide a replacement, the match can be delayed by 10 minutes at most.
+14. All players and the staff must be treated with respect. Instructions of the Tournament Management are to be followed. Decisions labeled as final are not to be objected.
+15. Disrupting the match by foul play, insulting and provoking other players or the staff, delaying the match, or other deliberate inappropriate misbehavior is strictly prohibited.
+16. Unexpected incidents are to be handled by the Tournament Management.
+17. Penalties for violating the tournament rules may include:
+    - Exclusion of specific players for one beatmap
+    - Exclusion of specific players for an entire match
+    - Declaring the match as Lost by Default
+    - Disqualification from the entire tournament
+    - Disqualification from the current and future official tournaments until appealed
+    - Any modification of these rules will be announced.
+
+### Tournament registration
+
+1. In order to be registered for the tournament, a representative of each team will have to propose a team roster comprising of 3-4 players in the discussion thread following a template set by the Tournament Management as follows:
+
+```
+Team Name: (...)
+Region: (...)
+
+Player 1 (Captain): (...)
+Player 2: (...)
+Player 3: (...)
+Player 4: (...)
+
+Time Zone: (...)
+```
+
+2. To ensure valid and serious registrations, all proposed team rosters will be checked by the Tournament Management.
+3. All accepted teams will be announced in the discussion thread after the Registration Phase.
+   - **A team should have at least three players registered in order to be eligible to participate.**
+4. Once a team roster has been accepted, it remains locked for the rest of the tournament. No further changes may be made to it except on extraordinary circumstances under the Tournament Management's approval.
+
+### Match instructions
+
+1. A member of the Tournament Management will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!catch, Team-Vs., Win Condition: 'Score'. Room name must be "CWC: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
```

Comment:
```suggestion
   - The team mentioned first in the room must be named as the "red team"; the team mentioned second in the room must be named as the "blue team."
```

Reply:
will think of sth else i guess since i get your point but "room name" is already the proper term here